lib/salesforce/client.rb

```diff
@@ -0,0 +1,24 @@
+# Eventually, this should be where we have code to talk directly to salesforce
+# instead of using the openstax_salesforce gem.
+# For now - this is just to get the salesforce client to interact with the pardot api
+
+module Salesforce
+  class Client
+    def initialize
+      salesforce_secrets = Rails.application.secrets.salesforce
+
+      salesforce_client = Restforce.new(username: salesforce_secrets[:username],
+                                        password: salesforce_secrets[:password],
+                                        security_token: salesforce_secrets[:security_token],
+                                        client_id: salesforce_secrets[:consumer_key],
+                                        client_secret: salesforce_secrets[:consumer_secret],
+                                        host: salesforce_secrets[:login_domain],
+                                        api_version: '41.0')
```

Comment:
make sure this is as late as possible to delay being forced to update sooner :-)